static/docs/commands-reference/import-url.md

```diff
@@ -284,6 +284,7 @@ $ tree
     ├── featurization.py
     ├── prepare.py
     └── train.py
+```
```

Comment:
This whole commit is a typo... Sry

Fixed in https://github.com/iterative/dvc.org/pull/522/commits/be18aad12fa66a917d45a8200fa72150b27a10d0#diff-0a462fb4a411db0aa4af5e0a1036b7f0L291-L293